spring-boot/proxy/src/main/java/eu/coatrack/proxy/metrics/MetricsCounterService.java

```diff
@@ -28,15 +28,21 @@
 import org.springframework.stereotype.Service;
 
 import java.sql.Date;
-import java.time.LocalDate;
+import java.time.*;
 import java.util.StringJoiner;
+import java.util.TimeZone;
 import java.util.UUID;
 
 import eu.coatrack.api.ApiKey;
 
 @Service
 public class MetricsCounterService {
 
+    // Quick fix of #237 by setting default time zones of Admin and Gateway to UCT timezone.
```

Comment:
This pull request changes only sets the Gateway timezone, but according to this comment it should also set the Admin timezone.